### PR TITLE
updating faucet version and some test integration

### PR DESCRIPTION
**Pull request by grafnu**

### CODE REVIEW

File: forch/faucet_state_collector.py [hunk 1]
```diff
@@ -531,21 +531,22 @@ def process_port_state(self, timestamp, name, port, state):
...
+    def process_lag_state(self, timestamp, name, port, lacp_state):
         """process lag change event"""
         with self.lock:
             egress_state = self.topo_state.setdefault('egress', {})
             old_egress_name = egress_state.get(EGRESS_DETAIL)
-            if old_egress_name and old_egress_name != name and not state:
+            if old_egress_name and old_egress_name != name and not lacp_state:
                 return
 
+            lacp_up = lacp_state == 2
```
Comment (didovesei) [inline forch/faucet_state_collector.py]: What is 2?

File: forch/faucet_state_collector.py [hunk 2]
```diff
@@ -531,21 +531,22 @@ def process_port_state(self, timestamp, name, port, state):
...
 
     @dump_states
-    def process_lag_state(self, timestamp, name, port, state):
+    def process_lag_state(self, timestamp, name, port, lacp_state):
         """process lag change event"""
         with self.lock:
             egress_state = self.topo_state.setdefault('egress', {})
             old_egress_name = egress_state.get(EGRESS_DETAIL)
-            if old_egress_name and old_egress_name != name and not state:
+            if old_egress_name and old_egress_name != name and not lacp_state:
```
Comment (didovesei) [inline forch/faucet_state_collector.py]: What is the possible values of lacp_state now?

Comment (grafnu) [reply forch/faucet_state_collector.py]: https://github.com/grafnu/faucet/blob/lacpstate/faucet/port.py#L29

**Comment by grafnu** [comment]
https://github.com/grafnu/faucet/blob/lacpstate/faucet/port.py#L29

File: forch/faucet_state_collector.py (hunk 2 above)
Comment (didovesei) [reply forch/faucet_state_collector.py]: Then I think this check:
 "not lacp_state"
should be changed to:
"lacp_state != 3"

The logic is if it is a lacp up state, then go ahead update the egress_state

**Pull request merged**